.github/workflows/test-os.yml

```diff
@@ -48,11 +49,33 @@ jobs:
         name: Install gotestsum
         run: |
           go install gotest.tools/gotestsum@${{ env.GOTESTSUM_VERSION }}
+      -
+        name: Install buildkitd.exe binary
+        run: |
+          go install .\cmd\buildkitd
+      -
+        name: Install containerd.exe binary
+        run: |
+          mkdir -p .\bin\installs -f
+          cd .\bin\installs
+          git clone https://github.com/containerd/containerd.git
+          cd containerd
+          go install .\cmd\...
+          cd ..\..\..
```

Comment:
Not sure if Github actions always starts from the "working directory" for each steps; but just doing this for explicitness.